### DISCUSSION TITLE
Easiest way to preview

### DISCUSSION THREAD

**Discussion by Meskat0** [3b1b/manim#2167] [Q&A]
Is it possible to use manim side view for ManimGL? or have any best idea to preview ?

**Comment by Splines** [comment]
- Grant shows his current workflow [in this new video](https://youtu.be/rbu7Zu5X1zI). You may also want to check the [Workflow section of the `videos` repository Readme](https://github.com/3b1b/videos?tab=readme-ov-file#workflow).
- If anyone has set up something similar for **VSCode** with all this nice interactivity and shortcuts, I'd be thrilled to hear about it! If you want to debug the Python script in VSCode, [this issue might be for you](https://github.com/3b1b/manim/issues/894).
- With regards to interactivity in Manim Community, see the discussion that I've opened [here](https://github.com/ManimCommunity/manim/discussions/3954).

**Reply by Splines** [reply]
Update: We're working on a VSCode extension called [**Manim Notebook**](https://github.com/Manim-Notebook/manim-notebook) to ease interactive preview.
